Gemfile

```diff
@@ -68,7 +68,7 @@ group :development, :test do
   gem 'factory_bot_rails'
   gem 'rubocop', '~> 0.52.0' # Pinned because upgrades require regenerating rubocop_todo.yml
   gem 'rainbow'
-  gem 'puma' # used by capybara 3
+  gem 'puma' unless ENV["DOCKER"] # used by capybara 3 and Docker
```

Comment:
```
    Failure/Error: raise LoadError, "Capybara is unable to load `puma` for its server, please add `puma` to your project or specify a different server via something like `Capybara.server = :webrick`."
     
     LoadError:
       Capybara is unable to load `puma` for its server, please add `puma` to your project or specify a different server via something like `Capybara.server = :webrick`.
     # /home/travis/build/fatfreecrm/fat_free_crm/vendor/bundle/ruby/2.4.0/gems/capybara-3.0.2/lib/capybara.rb:444:in `rescue in block in <main>'
     # /home/travis/build/fatfreecrm/fat_free_crm/vendor/bundle/ruby/2.4.0/gems/capybara-3.0.2/lib/capybara.rb:441:in `block in <main>'
     # /home/travis/build/fatfreecrm/fat_free_crm/vendor/bundle/ruby/2.4.0/gems/capybara-3.0.2/lib/capybara.rb:235:in `run_default_server'
     # /home/travis/build/fatfreecrm/fat_free_crm/vendor/bundle/ruby/2.4.0/gems/capybara-3.0.2/lib/capybara.rb:432:in `block in <main>'
     # /home/travis/build/fatfreecrm/fat_free_crm/vendor/bundle/ruby/2.4.0/gems/capybara-3.0.2/lib/capybara/server.rb:103:in `block in boot'
     # 
     #   Showing full backtrace because every line was filtered out.
     #   See docs for RSpec::Configuration#backtrace_exclusion_patterns and
     #   RSpec::Configuration#backtrace_inclusion_patterns for more information.
     # ------------------
     # --- Caused by: ---
     # LoadError:
     #   cannot load such file -- rack/handler/puma
     #   /home/travis/build/fatfreecrm/fat_free_crm/vendor/bundle/ruby/2.4.0/gems/bootsnap-1.3.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
```